src/ptrace.cc

```diff
@@ -47,6 +48,33 @@ void PtraceDetach(pid_t pid) {
   }
 }
 
+struct user_regs_struct PtraceGetRegs(pid_t pid) {
```

Comment:
Yes, this is purely a matter of ignorance on my part - this is literally the first thing I've written in C++.